nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-core/src/main/java/org/nuxeo/elasticsearch/commands/IndexingCommandsStacker.java

```diff
@@ -154,6 +158,10 @@ protected void stackCommand(DocumentModel doc, String eventId, boolean sync) {
         }
     }
 
+    private boolean indexIsLatestVersion() {
+        return ! Boolean.parseBoolean(Framework.getProperty(AbstractSession.DISABLED_ISLATESTVERSION_PROPERTY, "false"));
```

Comment:
No space after `!`.
And `Framework#isBooleanPropertyTrue(DISABLED_ISLATESTVERSION_PROPERTY)`?